src/util/persistent_logger.js

```diff
@@ -21,9 +21,9 @@ class PersistentLogger {
      * @param {string} dir parent directory
      * @param {string} namespace file prefix
      * @param {{
-     *  max_interval?: Number,
+     *  poll_interval?: Number,
      *  locking?: "SHARED" | "EXCLUSIVE",
-     *  disable_rotate?: boolean,
+     *  disable_poll_active?: boolean,
```

Comment:
I think we can remove this and rely on poll_interval only so we don't forget to set it by the various callers that only read the log or write a temp log.